openapi.json

```diff
@@ -0,0 +1,1413 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Fleet API",
+    "version": "2022-04"
+  },
+  "servers": [
+    {
+      "url": "/api/latest"
+    }
+  ],
+  "paths": {
+    "/fleet/hosts": {
+      "get": {
+        "operationId": "get_hosts",
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
```

Comment:
Considering this is auto-generated from the `jsonschema` package, this is actually quite good - the equivalent in our docs is here: https://fleetdm.com/docs/using-fleet/rest-api#list-hosts